packages/siopv2-oid4vp-op-auth/src/session/OpSession.ts

```diff
@@ -221,12 +220,13 @@ export class OpSession {
     if (!resolveOpts.subjectSyntaxTypesSupported || resolveOpts.subjectSyntaxTypesSupported.length === 0) {
       resolveOpts.subjectSyntaxTypesSupported = await this.getSupportedDIDMethods(true)
     }
+    //todo: populate with the right verification params. In did-auth-siop we don't have any test that actually passes this parameter
+    // and furthermore, it receives a couple of parameters that don't make that much sense for OP. like presentationVerificationCallback and revocationOpts
```

Comment:
This comment doesn't make any sense, as linked domains need to be verified on both ends

Reply:
removed the second part of the comment (`and furthermore...`).
the Verification object is now:
```
export interface Verification {
  presentationVerificationCallback?: PresentationVerificationCallback
  revocationOpts?: RevocationOpts
  replayRegistry?: IRPSessionManager
}
```